src/ResourceTile/ResourceTile.tsx

```diff
@@ -61,7 +61,7 @@ export interface Props {
   overlays?: React.ReactNode[];
   size: IconSize;
   title: string;
-  url: string;
+  url?: string;
```

Comment:
I don't think we have to, but just pointing out that if we wanted we could make the types extra fancy and require that at least one of `url` or `onClick` is provided (right now it's technically not a type error to omit both)

Reply:
yeah it works, I'll give the fancier option a try